### ISSUE TITLE
Pull down many selection does not stick

### ISSUE THREAD

**Issue by Evhelden** [PopcornTimeTV/PopcornTimeTV#294]
Pulled the new master.

The subtitles are (although the issue is closed) not yet in sync. 

Selecting the pull down menu and go for a delay does not work. Selected option does not stick in the subtitles section. This also counts for language and encoding options.

**Comment by r1qu3br** [comment]
I tested in the latest version and it worked perfectly, I just think it should increase a little more the sync limit 5.0 to -5.0 is still little

**Comment by r1qu3br** [comment]
Here also doesn't work, I'm using the Apple remote on iOS for the function

**Comment by Evhelden** [comment]
Do you get the subtitle delay working? I can't get it fixed.

Regards,

Erwin van Helden
+971569980582
Sent from iPhone

**Comment by r1qu3br** [comment]
The subtitles are working for me perfectly in the latest version, only the pull down is not working on the Harmony Control

**Comment by FlaviaBondJamesBondsHusband** [comment]
@Evhelden

> The subtitles are (although the issue is closed) not yet in sync.

We don't provide subtitles, we pull them from OpenSubtitles and it is up to their users to fix the subtitle sync. We do, however, offer an option to shift the subtitle delay by up to +/- 5s. You can find this option in the view controller presented when you swipe down on your apple tv remote.

**Comment by Evhelden** [comment]
Got that. Can not get the +/- 5s to work. 

Same subtitle on downloaded series in. Plex in sync.

Will try with with another movie and let you know.

**Comment by Evhelden** [comment]
Tried today with ' the accountant '  on 1080p. The subtitles are +2s early. I can not get the option of the pull down menu work. I can select it, but it does not change and when re-opening the pull down menu it shows again that the subitles are at +0.0. 

When comparing with Showbox on Android TV box and Cinemabox (CB) on Android those applications on Android have the subtitles in sync. Hence I do not believe it's the quality of the subtitles.

How to make sure I can get the subtitles in sync on PopCornTime on Apple TV? How to make sure the pull down menu works?

**Comment by FlaviaBondJamesBondsHusband** [comment]
> When comparing with Showbox on Android TV box and Cinemabox (CB) on Android those applications on Android have the subtitles in sync. Hence I do not believe it's the quality of the subtitles.

OK, first Showbox and Cinemabox are both closed source so there is no way of you checking that they use the same subtitle source as us. Second, they're using a completely different OS than this (Android vs tvOS) and likely a completely different media player SDK which makes that point completely irrelevant. 

> Tried today with ' the accountant ' on 1080p. The subtitles are +2s early. I can not get the option of the pull down menu work. I can select it, but it does not change and when re-opening the pull down menu it shows again that the subitles are at +0.0.

I just tried The Accountant in 1080p, both on-device and in sim, and subtitle delay was working. You either haven't pulled master or it is some other issue with your ATV.

**Comment by Evhelden** [comment]
Ok. I will build again and try. If I can't get it working right, I do not know what I am doing wrong. Let me try again in the coming days.

**Comment by Evhelden** [comment]
Flavia,

Compiled it again, but now on the iMac. Subtitle in Dutch on ‘ The accountant ‘ 1080p. Slight delay.
Tried with Marco Polo 2nd Serie Episode 1. Delay still much.
Comparing base is with Plex, self downloaded on NAS. Subtitles from OpenSubtitles. Performance good.

Delay of subtitles I can find, but it is not working. The selection of +/- 5s highlights and the ‘V’ sign appears, but when pushing it up again and later checking by pulling down, the delay is back to 0.0. This feature is not working. The Dutch subtitles I can only select via the ‘settings’ menu. Although the languages show in this same pull down, selecting subtitles do not work via the pull down, just like the subtitle delay. Maybe this feature is not working for others either.

I like PopCornTV. I wanted to share the test result. I wish I could program and help, but no skills available other than sharing test results.
